tool_quality_checks/sql_check_views.py

```diff
@@ -0,0 +1,1530 @@
+# -*- coding: utf-8 -*-
+
+sql_checks = {
+    "sql_completeness_manhole": """
+------------------- Putten -------------------------
+----------------------------------------------------
+-- Maaiveldhoogte aanwezig
+CREATE OR REPLACE VIEW {schema}.put_maaiveldniveau_leeg AS
```

Comment:
Is het makkelijker een dir met losse sql files (`sql_completeness_manhole.sql`) te hebben en 1 functie om ze in te lezen? Kijk maar even.

Reply:
@reinout nu maak ik gebruik van variabelen, zoals {schema}. Is dat dan nog steeds mogelijk?

Reply:
Ja, dat kan. Pseudocode:

    template = open(filename).read()
    sql = template.format(schema="bla bla")
    db_connection.run_this(sql)

Maar.... alleen doen als het handig voor je is, hoor.